tests/src/lv_test_conf_full.h

```diff
@@ -1,5 +1,6 @@
 #define LV_MEM_SIZE                     (32 * 1024 * 1024)
 #define LV_DRAW_SW_SHADOW_CACHE_SIZE    8
+#define LV_DRAW_THREAD_STACKSIZE    (128 * 1024)
```

Comment:
Let's use the recommended 32kB in default config instead.

Reply:
Using a 32kb stack will cause a crash, as Thorvg consumes a significant amount of stack memory.

Reply:
Using a 32kb stack will cause a crash, as Thorvg consumes a significant amount of stack memory

Reply:
![20240321-202757](https://github.com/lvgl/lvgl/assets/103997543/c7b5625d-2eee-4538-b554-9cdb598a0073)